File: Polymorphism/Poly.py
class Superman:
    def fly(self):
        print("Superman")


class Plane:
    def fly(self):
        print("Plane")

class Mosquito:
    def fly(self):
        print("Mosquito")


flyingObjects = [Superman(), Plane(), Mosquito()]

for obj in flyingObjects:
    obj.fly()

File: Polymorphism/Poly.js
class Superman {
    fly() {
        console.log("Flying superman")
    }
}

class Plane {
    fly() {
        console.log("flying Plane")
    }
}

class Mosquito {
    fly() {
        console.log("flying mosquito")
    }
}

const flyingObjects = [new Superman(), new Plane(), new Mosquito()]

for (const obj of flyingObjects) {
    obj.fly()
}
